test/integration/openid4ci_test.go

```diff
@@ -27,54 +31,80 @@ import (
 // echo '127.0.0.1 vc-rest-echo.trustbloc.local' | sudo tee -a /etc/hosts
 
 func TestOpenID4CIFullFlow(t *testing.T) {
+	type test struct {
+		issuerProfileID string
+		issuerDIDMethod string
+		walletDIDMethod string
+	}
+
+	tests := []test{
+		{issuerProfileID: "bank_issuer", issuerDIDMethod: "orb", walletDIDMethod: "ion"},
+		{issuerProfileID: "did_ion_issuer", issuerDIDMethod: "ion", walletDIDMethod: "key"},
+	}
+
 	oidc4ciSetup, err := oidc4ci.NewSetup(testenv.NewHttpRequest())
 	require.NoError(t, err)
 
 	err = oidc4ciSetup.AuthorizeIssuerBypassAuth("test_org")
 	require.NoError(t, err)
 
-	initiateIssuanceURL, err := oidc4ciSetup.InitiatePreAuthorizedIssuance("bank_issuer")
-	require.NoError(t, err)
+	for _, tc := range tests {
+		fmt.Println(fmt.Sprintf("running tests with : issuerProfileID=%s issuerDIDMethod=%s walletDIDMethod=%s",
```

Comment:
Shouldn't be a space between `with` and `:`

Reply:
I removed the colon, as it's clearer without it.